sklearn/utils/arpack.py

```diff
@@ -42,10 +42,7 @@
 # ARPACK and handle shifted and shift-inverse computations
 # for eigenvalues by providing a shift (sigma) and a solver.
 
-__docformat__ = "restructuredtext en"
-
-__all__ = ['eigs', 'eigsh', 'svds', 'ArpackError', 'ArpackNoConvergence']
-import warnings
+from __future__ import division, print_function, absolute_import
```

Comment:
Do we need these imports? I guess it is OK but I worry about breaking other functions accidentally